<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="img_src">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>120</y>
      <width>191</width>
      <height>291</height>
     </rect>
    </property>
    <property name="text">
     <string>src</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="img_ref">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>120</y>
      <width>191</width>
      <height>291</height>
     </rect>
    </property>
    <property name="text">
     <string>ref</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="img_result">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>120</y>
      <width>191</width>
      <height>291</height>
     </rect>
    </property>
    <property name="text">
     <string>result</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="select_src">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>430</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>select_src</string>
    </property>
   </widget>
   <widget class="QPushButton" name="select_ref">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>430</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>select_ref</string>
    </property>
   </widget>
   <widget class="QPushButton" name="merge">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>430</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>합        성</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>select_src</sender>
   <signal>clicked()</signal>
   <receiver>img_src</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>473</y>
    </hint>
    <hint type="destinationlabel">
     <x>167</x>
     <y>338</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
